iap_watcher/iap_watcher_test.py

```diff
@@ -116,6 +124,55 @@ def testDisableEnable(self):
       loop_watcher=False)
     self.assertTrue(os.path.isfile(path))
 
+  def testEmptyMetadata(self):
+    """Tests that the state file does not exist if the metadata is empty."""
+    path = '%s/tmp' % self.pathname_
+    if os.path.isfile(path):
```

Comment:
should we put these cleanup calls in a setup method?

Reply:
Done. Yeah, I had that thought and forgot about it :), thanks for catching!